src/tests/suite/daos_obj.c

```diff
@@ -3437,6 +3592,10 @@ static const struct CMUnitTest io_tests[] = {
 	  split_sgl_update_fetch, async_disable, test_case_teardown},
 	{ "IO36: trigger server pool map refresh",
 	  io_pool_map_refresh_trigger, async_disable, test_case_teardown},
+	{ "IO37: Fetch nonexistent keys",
```

Comment:
I think those 2 tests can be just merged into 1 that tests a mixed usage with different key types.

Reply:
ok will merge both tests.